website/src/config-reference.md

```diff
@@ -37,16 +37,17 @@ vendor/bin/phpstan analyse -c phpstan.neon
 
 When using a config file, you have to pass the `--level|-l` option to `analyse` command (default value `0` does not apply here), or provide it as a `level` parameter in the config file itself. [Learn more about other command line options »](/user-guide/command-line-usage)
 
-If you do not provide a config file explicitly, PHPStan will look for files named `phpstan.neon` or `phpstan.neon.dist` in current directory.
+If you do not provide a config file explicitly, PHPStan will look for files named `phpstan.neon`, `phpstan.neon.dist`, or `phpstan.dist.neon` in the current directory.
```

Comment:
Needs to be specified that `phpstan.dist.neon` is supported since PHPStan 1.4.7.